client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -33,7 +33,7 @@ const DomainRegistrationSuggestion = React.createClass( {
 		let buttonClasses, buttonContent;
 
 		if ( suggestion.domain_name ) {
-			const newTLDs = [];
+			const newTLDs = [ '.uk' ];
```

Comment:
The immediate goal is `.co.uk`, and we're sizing up `.uk` behind that. I was thinking this would pick up both, but at this stage it's really just so that I can find it easily when I'm ready :)